src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { useMemo } from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The logo source of the asset
+     */
+    logoSrc?: string;
+    /**
+     * The name of the asset.
+     */
+    name?: string;
+    /**
+     * The symbol of the asset.
+     */
+    symbol?: string;
```

Comment:
We might want to set some of these properties as required, I can't see a version of the component on figma without the `name` or `symbol` properties. Also, when the fiatPrice property is not set, the design displays a "Unknown" text instead of "0"